akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedBehaviorImpl.scala

```diff
@@ -90,25 +76,28 @@ private[akka] final case class EventSourcedBehaviorImpl[Command, Event, State](
     // stashState outside supervise because StashState should survive restarts due to persist failures
     val stashState = new StashState(settings)
 
+    val actualSignalHandler: PartialFunction[Signal, Unit] = signalHandler match {
+      case Some(userProvided) ⇒ userProvided
```

Comment:
`.orElse(ourDefaultSIgnalHandler)` so that logging doesn't disappear just because user handles some other signal

Reply:
Hmm, didn't think of that. Need to figure out how to keep the defaults for Java as well then, did this same behavior there.